### ISSUE TITLE
Home busy spinner

### ISSUE THREAD

**Issue by DJMix-I-Can** [jurialmunkey/skin.arctic.zephyr.2#131]
Hey Jurialmunkey, the latest update made the busy spinner visible on the home screen again.
Thanks

**Comment by jurialmunkey** [comment]
Can you test with this version?
https://github.com/jurialmunkey/skin.arctic.zephyr.2/releases/tag/v0.9.53-alpha4

I think it should be fixed now.

**Comment by DJMix-I-Can** [comment]
Unfortunately, it didn't fix it.

**Comment by jurialmunkey** [comment]
Yep, you're right. Not sure why Container.NumItems is set to 0 when updating when clearly there are items in the container.

Anyway, I think I've finally figured out a way to fix it. Can you test this version and confirm:
https://github.com/jurialmunkey/skin.arctic.zephyr.2/releases/tag/v0.9.53-alpha5

**Comment by DJMix-I-Can** [comment]
That did it. Thanks again...

**Comment by jurialmunkey** [comment]
No worries! It was bugging me too, so I'm glad I finally found a fix!

**Comment by drinfernoo** [comment]
It seems like the latest version makes the busy spinner not show up even when widgets *are* loading?

**Comment by DJMix-I-Can** [comment]
When you first open Kodi, focus is on the menu & you won't see the spinner. Once you scroll down (focus) to your widgets, you will see the spinner. Once everything is loaded, you won't see the spinner again until you reopen Kodi. That's how it's performing on mine, which I'm ok with.

**Comment by jurialmunkey** [comment]
Yeah this spinner issue is a pain. I'm not sure why my visibility conditions aren't being followed correctly. Re-opening this issue until I find a fix.

**Comment by tekknight** [comment]
I also want to mention that there is an unwanted busy spinner when using a startup video. I hope you can fix that also.
